src/core/EntrySearcher.cpp

```diff
@@ -136,11 +137,21 @@ void EntrySearcher::setCaseSensitive(bool state)
     m_caseSensitive = state;
 }
 
-bool EntrySearcher::isCaseSensitive()
+bool EntrySearcher::isCaseSensitive() const
 {
     return m_caseSensitive;
 }
 
+void EntrySearcher::setSkipProtected(bool skip)
+{
+    m_skipProtected = skip;
+}
+
+bool EntrySearcher::isSkipProtected() const
+{
+    return m_skipProtected;
+}
+
```

Comment:
Either set it in the constructor (persistent) or via function (non-persistent), but don't support both. EntrySearcher is meant to be one-time-use object. Recommend removing these.

Reply:
I made the necessary changes